subprojects/distributions-dependencies/build.gradle.kts

```diff
@@ -106,7 +106,7 @@ dependencies {
         api(libs.junit)                 { version { strictly("4.13.2") }}
         api(libs.junit5JupiterApi)      { version { strictly("5.7.2") }}
         api(libs.junit5Vintage)         { version { strictly("5.7.2") }}
-        api(libs.junitPlatform)         { version { strictly("1.7.2") }}
+        api(libs.junitPlatform)         { version { strictly("1.8.1") }}
```

Comment:
Why are you managing the versions separately here? Consider using the `junit-bom` to get properly aligned versions. The JUnit Platform version should match the Jupiter version if you ignore the major version.

Reply:
I don't know why we ended up with this setup. Could you give me a diff that'd make this more correct/simpler?

Reply:
I couldn't figure out where the `libs` definitions come from, but I would assume you could to something like `api(platform(libs.junitPlatformBom)) {...}` for https://search.maven.org/artifact/org.junit/junit-bom/5.8.2/pom 

If it is possible to use `platform` in dependency constraints.

Reply:
Thanks. Let's do this in a separate PR.